assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -157,8 +216,22 @@ function FontPickerContainer({ value, onSelect, onClose }) {
 
   return (
     <PickerContainer ref={ref}>
+      <SearchInput
```

Comment:
Should we perhaps auto-focus this input when the user opens the dropdown? But still with the currently selected font in focus until input happens of course.

Reply:
You mean focus the input but have the font selected? Not focused. Two items cannot be focused at once.